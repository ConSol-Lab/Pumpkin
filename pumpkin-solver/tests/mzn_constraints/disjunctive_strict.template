predicate gecode_schedule_unary(array [int] of var int: x,array [int] of int: p);

var 0..8: x :: output_var;
var 0..8: y :: output_var;
var 0..8: z :: output_var;

constraint gecode_schedule_unary([x, y, z], [2, 3, 5]);

solve satisfy;
