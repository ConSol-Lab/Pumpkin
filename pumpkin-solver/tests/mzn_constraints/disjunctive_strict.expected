x = 8;
y = 5;
z = 0;
----------
x = 5;
y = 7;
z = 0;
----------
x = 5;
y = 8;
z = 0;
----------
x = 6;
y = 8;
z = 0;
----------
x = 6;
y = 8;
z = 1;
----------
x = 0;
y = 7;
z = 2;
----------
x = 0;
y = 8;
z = 2;
----------
x = 8;
y = 0;
z = 3;
----------
x = 0;
y = 8;
z = 3;
----------
x = 1;
y = 8;
z = 3;
----------
x = 3;
y = 0;
z = 5;
----------
x = 0;
y = 2;
z = 5;
----------
x = 3;
y = 0;
z = 6;
----------
x = 4;
y = 0;
z = 6;
----------
x = 4;
y = 1;
z = 6;
----------
x = 0;
y = 2;
z = 6;
----------
x = 0;
y = 3;
z = 6;
----------
x = 1;
y = 3;
z = 6;
----------
x = 3;
y = 0;
z = 7;
----------
x = 4;
y = 0;
z = 7;
----------
x = 5;
y = 0;
z = 7;
----------
x = 4;
y = 1;
z = 7;
----------
x = 5;
y = 1;
z = 7;
----------
x = 0;
y = 2;
z = 7;
----------
x = 5;
y = 2;
z = 7;
----------
x = 0;
y = 3;
z = 7;
----------
x = 1;
y = 3;
z = 7;
----------
x = 0;
y = 4;
z = 7;
----------
x = 1;
y = 4;
z = 7;
----------
x = 2;
y = 4;
z = 7;
----------
x = 3;
y = 0;
z = 8;
----------
x = 4;
y = 0;
z = 8;
----------
x = 5;
y = 0;
z = 8;
----------
x = 6;
y = 0;
z = 8;
----------
x = 4;
y = 1;
z = 8;
----------
x = 5;
y = 1;
z = 8;
----------
x = 6;
y = 1;
z = 8;
----------
x = 0;
y = 2;
z = 8;
----------
x = 5;
y = 2;
z = 8;
----------
x = 6;
y = 2;
z = 8;
----------
x = 0;
y = 3;
z = 8;
----------
x = 1;
y = 3;
z = 8;
----------
x = 6;
y = 3;
z = 8;
----------
x = 0;
y = 4;
z = 8;
----------
x = 1;
y = 4;
z = 8;
----------
x = 2;
y = 4;
z = 8;
----------
x = 0;
y = 5;
z = 8;
----------
x = 1;
y = 5;
z = 8;
----------
x = 2;
y = 5;
z = 8;
----------
x = 3;
y = 5;
z = 8;
----------
==========
